File: sum-of-digits.py
numbers = input("Enter list of numbers separated by comma: ")
separated_numbers = numbers.split(",")
converted_numbers = []

# Converting each string number to number
for item in separated_numbers:
    converted_numbers.append(float(item))

# Sum of the entire array
sum = 0
for item in converted_numbers:
    sum += item

# Printing the sum
print(f"Total sum of {numbers} is {sum}")

File: sum-of-digits.js
const input = require("./input");

(async() => {
    const numbers = await input("Enter list of numbers separated by comma: ");
    const separated_numbers = numbers.split(",");
    let converted_numbers = [];
    
    // Converting each number string to number
    separated_numbers.map((item) => {
        converted_numbers.push(parseFloat(item));
    })

    // Sum of the entire array
    let sum = 0;
    converted_numbers.forEach(item => {
        sum += item;
    });
    
    // Printing the sum
    console.log(`Total sum of ${numbers} is ${sum}`);
})();